src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -72,6 +72,33 @@
  * <pre>
  * &lt;module name=&quot;MissingOverride&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test extends SuperClass {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     &#64;Override
+ *     public void test1() { // OK
+ *
+ *     }
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void test2() { // violation, not be annotated with &#64;Override
```

Comment:
` not be` I guess it should be `should not be`

Reply:
sorry, `not be` is incorrect, which causes a mistake. I think `should be annotated with &#64;Override` is correct and is what I want to express